extraction/src/it/scala/org/broadinstitute/monster/encode/extraction/ExtractionPipelineIntegrationSpec.scala

```diff
@@ -0,0 +1,84 @@
+package org.broadinstitute.monster.encode.extraction
+
+import better.files.File
+import org.broadinstitute.monster.common.PipelineBuilderSpec
+
+class ExtractionPipelineIntegrationSpec extends PipelineBuilderSpec[Args] {
+  val outputDir = File.newTemporaryDirectory()
+  override def afterAll(): Unit = outputDir.delete()
+
+  val testBatchSize: Long = 100
+
+  val allParams: List[(String, String)] = {
+    List(
+      // Default Human Params
+      "organism.name" -> "human",
+      // Human Test Sample
+      "accession" -> "ENCBS488IUR"
+    )
+  }
+
+  /// KICK OFF EXTRACTION
+  override val testArgs =
+    Args(outputDir.pathAsString, testBatchSize, allParams)
+  override val builder = ExtractionPipeline.pipelineBuilder
+
+  // Paths for entity files downloaded
+  val AnalysisStepPath = (outputDir.pathAsString + "/AnalysisStep")
+  val AnalysisStepRunPath = (outputDir.pathAsString + "/AnalysisStepRun")
+  val AnalysisStepVersionPath = (outputDir.pathAsString + "/AnalysisStepVersion")
+  val AntibodyLotPath = (outputDir.pathAsString + "/AntibodyLot")
+  val BiosamplePath = (outputDir.pathAsString + "/Biosample")
+  val DonorPath = (outputDir.pathAsString + "/Donor")
+  val ExperimentPath = (outputDir.pathAsString + "/Experiment")
+  val FilePath = (outputDir.pathAsString + "/File")
+  val FunctionalCharacterizationExperimentPath = (outputDir.pathAsString + "/FunctionalCharacterizationExperiment")
+  val LibraryPath = (outputDir.pathAsString + "/Library")
+  val ReplicatePath = (outputDir.pathAsString + "/Replicate")
+  val TargetPath = (outputDir.pathAsString + "/Target")
```

Comment:
Style nitpick: it'd be more conventional for these to all be lower-camel-case, i.e.
```suggestion
  val targetPath = (outputDir.pathAsString + "/Target")
```
Upper-camel-case is typically reserved for static constants